editor/components/upload/upload.ts

```diff
@@ -17,7 +17,7 @@ export class Upload {
 
     handleFileChange(event: Event) {
         let files = (event.target as HTMLInputElement).files;
-        let file: File = files && files[0];
+        let file = files && files[0];
```

Comment:
Why do you drop the type annotation here? Is it because it is no longer possible in the new versions of our dependencies or because you thank it was obvious?